gddo-server/config.go

```diff
@@ -173,6 +180,7 @@ func buildFlags() *pflag.FlagSet {
 	flags.String(ConfigGAERemoteAPI, "", "Remoteapi endpoint for App Engine Search. Defaults to serviceproxy-dot-${project}.appspot.com.")
 	flags.Float64(ConfigTraceSamplerFraction, 0.1, "Fraction of the requests sampled by the trace API.")
 	flags.Float64(ConfigTraceSamplerMaxQPS, 5, "Max number of requests sampled every second by the trace API.")
+	flags.String(ConfigLogLevel, "info", "Determine which level of logs to print.")
```

Comment:
maybe put acceptable values in the help string?  `(info, warning, etc.)`.

Reply:
Done!